app/lib/methods/subscriptions/rooms.ts

```diff
@@ -241,6 +241,7 @@ const getRoomQueueId = (rid: string) => `ROOM-${rid}`;
 
 const debouncedUpdate = (subscription: ISubscription) => {
 	if (!subTimer) {
+		// @ts-ignore
 		subTimer = setTimeout(() => {
```

Comment:
Can you try `ReturnType<typeof setTimeout>`?
https://twitter.com/wesbos/status/1532040298246942723